FFCDemoPaymentService.Tests/MessagingTests/UnitTest1.cs

```diff
@@ -0,0 +1,90 @@
+using System;
```

Comment:
The name of the file should match the name of the class it contains and be meaningful.  Ie. not `UnitTest1.cs`

Reply:
removed file